File: Examples/VisualBasic/WorkingWithTasks/WriteTaskCalendar.vb
Imports Aspose.Tasks.Saving

'
'This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Tasks for .NET API reference 
'when the project is build. Please check https://docs.nuget.org/consume/nuget-faq for more information. 
'If you do not wish to use NuGet, you can manually download Aspose.Tasks for .NET API from http://www.aspose.com/downloads, 
'install it and then add its reference to this project. For any issues, questions or suggestions 
'please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace WorkingWithTasks
    Public Class WriteTaskCalendar
        Public Shared Sub Run()
            ' ExStart:WriteTaskCalendar
            ' Create project instance
            Dim project As New Project()

            ' Add task
            Dim tsk1 As Task = project.RootTask.Children.Add("Task1")

            ' Create calendar and assign to task
            Dim cal As Aspose.Tasks.Calendar = project.Calendars.Add("TaskCal1")
            tsk1.[Set](Tsk.Calendar, cal)
            ' ExEnd:WriteTaskCalendar
        End Sub
    End Class
End Namespace

File: Examples/CSharp/WorkingWithTasks/WriteTaskCalendar.cs
/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Tasks for .NET API reference 
when the project is build. Please check https:// Docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Tasks for .NET API from https://www.nuget.org/packages/Aspose.Tasks/, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using https://forum.aspose.com/c/tasks
*/

namespace Aspose.Tasks.Examples.CSharp.WorkingWithTasks
{
    internal class WriteTaskCalendar
    {
        public static void Run()
        {
            //ExStart:WriteTaskCalendar
            //ExFor: Tsk.Calendar
            //ExSummary: Shows how to set a task calendar.
            // Create project instance
            var project = new Project();

            // Add task
            var task = project.RootTask.Children.Add("Task1");

            // Create calendar and assign to task
            var calendar = project.Calendars.Add("TaskCal1");
            task.Set(Tsk.Calendar, calendar);
            //ExEnd:WriteTaskCalendar      
        }
    }
}
